internal/state/indexer/sink/psql/schema.sql

```diff
@@ -28,7 +28,7 @@ CREATE TABLE tx_results (
   rowid BIGSERIAL PRIMARY KEY,
 
   -- The block to which this transaction belongs.
-  block_id BIGINT NOT NULL REFERENCES blocks(rowid),
+  block_id BIGINT NOT NULL REFERENCES blocks(height),
```

Comment:
changed these because I was talking with a data scientist about this data and he could reason about it better like this

Reply:
This isn't relevant to the PR, let's consider that change separately. The schema is currently based on rowids, so we should have a better reason for changing it than one person's casual notion. (Maybe there is a better reason but let's write that down and make sure we update things consistently for the change if so)